.gitignore

```diff
@@ -139,4 +139,6 @@ jscomp/outcome_printer/reason_syntax_util.ml
 jscomp/outcome_printer/reason_syntax_util.mli
 jscomp/bin/all_ounit_tests.ml
 native/
-vendor/ocaml
```

Comment:
it is not ignored currently, due to the fact we need such file when publishing a release to npm

Reply:
:ok: